aikiblog/models.py

```diff
@@ -149,3 +149,14 @@ def save(self, force_insert=False, force_update=False, using=None):
         dt = '{:%Y-%m-%d %H-%M}'.format(d)
         self.slug = slugify(str(self.user) + " " + str(dt) + " " + str(self.place))
         super(Training, self).save(force_insert, force_update, using)
+
+
+class TrainingComment(models.Model):
+    training_id = models.ForeignKey(Training)
```

Comment:
To pole powinno się nazywać training, bo uderzasz tak faktycznie do obiektu, a nei do jego pola. Poprawię to, bo widzę, że nie ma migracji. Dorobię migracje. Migracje muszą być!